src/sssom_schema/schema/sssom_schema.yaml

```
@@ -584,6 +584,38 @@ slots:
         description: A one-to-one mapping.
       - value: "1:n"
         description: A one-to-many mapping.
+    see_also:
+      - https://github.com/mapping-commons/sssom/blob/master/examples/schema/cardinality-scope-empty.sssom.tsv
+  cardinality_scope:
+    description: A list of mapping slots that define the scope for the value found
+      in the mapping_cardinality slot. Mappings are considered to belong to the same
+      scope if they have the same value for all slots listed in the scope. If no
+      scope is defined, the default scope is empty, meaning that all mappings belong
+      to a single scope that is identical to the entire mapping set.
+      The behaviour if a value in the list does not correspond to a valid slot name
+      is undefined.
+    range: string
+    multivalued: true
+    instantiates:
+      - sssom:Propagatable
+      - sssom:Versionable
+    annotations:
+      propagated: true
+      added_in: "1.1"
+    examples:
+      - value:
+          - "predicate_id"
+        description: Indicates that mapping_cardinality is computed relatively to all
+          mappings that have the same predicate.
+      - value:
+          - "predicate_id"
+          - "object_source"
```

I do not myself know if this syntax is valid, or you need to use something more like 

```suggestion
          - ["object_source", "predicate_id"]
```

But since the qc is not complaining, lets let it stand.